esp/public/media/theme_editor/less/variables.less

```diff
@@ -208,8 +208,8 @@
 
 // Navbar
 // -------------------------
-@navbarCollapseWidth:             979px;
-@navbarCollapseDesktopWidth:      @navbarCollapseWidth + 1;
+//@navbarCollapseWidth:             979px;
+//@navbarCollapseDesktopWidth:      @navbarCollapseWidth + 1;
```

Comment:
We can probably keep this one uncommented and remove it from variables_custom.less, since I don't think we'll ever want to modify both variables.